main.go

```diff
@@ -130,18 +138,23 @@ func doIncrementalMode(es *elasticsearch.Elasticer, d *database.Databaser, clien
 
 	go client.Listen()
 
-	client.AddConsumer(messaging.IncrementalExchange, "direct", "templeton.incremental", messaging.IncrementalKey, func(del amqp.Delivery) {
```

Comment:
Probably on your radar, but make sure to clean up the now-unused stuff in messaging at some point (`messaging.IncrementalExchange` here, but there's probably others)